packages/kbn-expandable-flyout/src/actions.ts

```diff
@@ -39,6 +39,10 @@ export const openPanelsAction = createAction<{
    * Unique identifier for the flyout (either the urlKey or 'memory')
    */
   id: string;
+  /**
+   * Whether or not the panel should sync to url state
+   */
+  shouldSync?: boolean;
```

Comment:
We'll need to re-consider how we handle url state storage as it currently syncs the `params` object, which can get large